### PR TITLE
Update namespace from Twiml to TwiML

### PR DESCRIPTION
**Pull request by bermudezmt**
Signed-off-by: Maria Bermudez <bermudez.mt@gmail.com>

As pointed out by a customer and Sarah via https://github.com/TwilioDevEd/api-snippets/pull/883#issuecomment-805173740, the namespace should be `TwiML` instead of what we had which was `Twiml`. @sarahcstringer , can you PTAL?

Thanks!

### CODE REVIEW

File: tools/automation/twiml-generator/generator.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def generate_code_sample_filepath(twiml_filepath, language):
                     twiml_filepath, language
                 )
                 print("{} -> {}".format(twiml_filepath, code_filepath), end='')
-                code_generator = TwimlCodeGenerator(
+                code_generator = TwiMLCodeGenerator(
```
Comment (bermudezmt) [inline tools/automation/twiml-generator/generator.py:66]: Oh, just noticed these. Let me push another commit.

Comment (bermudezmt) [reply tools/automation/twiml-generator/generator.py]: Fixed via https://github.com/TwilioDevEd/api-snippets/pull/885/commits/2a37f99db02e99ebfbab1c2aa3d2b14deebd7048

**Comment by bermudezmt** [comment]
FYI, @maylonpedroso , I'm seeing this warning on the terminal:

```
GitHub found 9 vulnerabilities on TwilioDevEd/api-snippets's default branch (3 high, 2 moderate, 4 low). To find out more, visit: https://github.com/TwilioDevEd/api-snippets/security/dependabot
```

**Review by maylonpedroso — APPROVED** [review]
:+1:

**Pull request merged**